components/AudioSettings/MuteButton.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+
+export default function MuteButton({ isMuted, setIsMuted, setCurrentVolume }) {
+  function toggleMute() {
+    const toggle = isMuted ? 0.3 : 0;
```

Comment:
Renaming this to volume would make it clearer what is safed in the variable

```suggestion
    const volume = isMuted ? 0.3 : 0;
```